EpicLoot/LootRoller.cs

```diff
@@ -196,15 +194,13 @@ private static List<GameObject> RollLootTableInternal(LootTable lootTable, int l
             {
                 return results;
             }
-
```

Comment:
Why did you take out these blank lines? Blank lines are useful to create breaks between groups. The comment I made to you earlier was for extraneous line breaks (like at the beginning or end of a block) and double line breaks.

Reply:
Put spaces back in. :)